### PR TITLE
fix: fixed broken footer links for all non-finnish languages

### PR DESCRIPTION
**Pull request by arpargo**
Footer links were broken on non-finnish languages.

Footer links have been updated to support non-finnish languages.

### CODE REVIEW

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1019-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://kerrokantasi-pr1019-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1019-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://kerrokantasi-pr1019-ui.dev.hel.ninja :laughing::tada::tada::tada:

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -8,11 +8,11 @@ import settings from '@city-assets/settings.json';
 import { Footer as HDSFooter, Logo } from 'hds-react';
```
Comment (mikkojamG) [inline src/components/Footer.jsx:8]: Unused import above `import urls from '@city-assets/urls.json';`

**Review by mikkojamG — APPROVED** [review]

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1019-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://kerrokantasi-pr1019-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1019-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://kerrokantasi-pr1019-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Review by mikkojamG — APPROVED** [review]

**Pull request merged**